app/src/main/java/org/fossasia/susi/ai/chat/STTfragment.kt

```diff
@@ -28,13 +28,14 @@ class STTfragment : Fragment() {
 
     override fun onAttach(context: Context?) {
         super.onAttach(context)
-        chatPresenter = ChatPresenter(activity as ChatActivity)
+        chatPresenter = ChatPresenter(requireContext())
     }
 
     @NonNull
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
         val rootView = inflater.inflate(R.layout.fragment_sttframe, container, false)
-        (activity as ChatActivity).fabsetting.hide()
+        val thisActivity = activity
+        if (thisActivity is ChatActivity) thisActivity.fabsetting.hide()
```

Comment:
Make this a lateinit var

Reply:
If I make thisActivity lateinit var, it should be non null type. But activity is always null type and also cannot smart cast to activity.